server/server.go

```diff
@@ -19,6 +19,7 @@ const (
 	refreshQueueSize  = 2048
 )
 
+// Server is a caching dns proxy that upgrades DNS to DNS over TLS.
```

Comment:
Nit: Make "dns" uppercase. Use "DNS-over-TLS" for coherence.

Reply:
Done.